distributed/tests/test_client.py

```diff
@@ -3787,7 +3787,6 @@ async def test_reconnect_timeout(c, s):
     ) as logger:
         await s.close()
         while c.status != "closed":
-            await c._update_scheduler_info()
```

Comment:
I assume these should ensure that while the scheduler is closing this particular call is silent. I don't think this is how we should test it. 
These tests failed because the update_scheduler_info is basically a RPC which is cancelled since the client is closing. This cancellation will then propagate to the test itself and let it fail as what appears to be a `TimeoutError`